flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/util/SegmentsUtil.java

```diff
@@ -417,7 +415,7 @@ private static boolean inFirstSegment(MemorySegment[] segments, int offset, int
 	 * @param index bit index from base offset.
 	 */
 	public static void bitUnSet(MemorySegment segment, int baseOffset, int index) {
-		int offset = baseOffset + ((index & BIT_BYTE_POSITION_MASK) >>> 3);
+		int offset = baseOffset + (index >>> 3);
 		byte current = segment.get(offset);
 		current &= ~(1 << (index & BIT_BYTE_INDEX_MASK));
```

Comment:
It seems that `BIT_BYTE_INDEX_MASK` is not needed either.

Reply:
Good point. Thanks for your comments.

Reply:
Can you do it in this PR?